src/games/strategy/common/ui/BasicGameMenuBar.java

```diff
@@ -172,10 +171,8 @@ protected void addGameNotesMenu(final JMenu parentMenu) {
     if (notesProperty != null && notesProperty.trim().length() != 0) {
       final String notes = LocalizeHTML.localizeImgLinksInHTML(notesProperty.trim());
       m_gameNotesPane = new SoftJEditorPane(notes);
-      parentMenu.add(SwingAction.of("Game Notes...", e -> {
-        SwingUtilities.invokeLater(new Runnable() {
-          @Override
-          public void run() {
+      parentMenu.add(SwingAction.of("Game Notes...", e ->
+          SwingUtilities.invokeLater(() -> {
```

Comment:
The anonymous inner class version is more difficult IMO. The really brutal stuff is why a SwingAction is being adding to a parentMenu, doesn't make sense. Swing names and conventions could have been better, pretty amazingly old stuff.